index.bs

```diff
@@ -6604,10 +6607,10 @@ of user tracking:
     User agents may require the user to authorize access to databases
     before a site can use the feature.
 
-: Origin-tracking of stored data
+: Storage Key tracking of stored data
```

Comment:
This section was confusing before, and I don't think this improves the situation. Maybe rename to "Attribution of third-party storage" ?